problems/[230619] Top K Frequent Words.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/top-k-frequent-words
+ *
+ * @param {string[]} words
+ * @param {number} k
+ * @return {string[]}
+ */
+const topKFrequent = function(words, k) {
+  const map = {};
+
+  words.forEach((word) => {
+    if (map[word]) return map[word] += 1;
+      map[word] = 1;
+  });
+
+  return Object.keys(map).sort().sort((wordA, wordB) => {
+    return map[wordB] - map[wordA];
+  }).splice(0,k); 
```

Comment:
```suggestion
  return Object.keys(map)
    .sort()
    .sort((wordA, wordB) => map[wordB] - map[wordA])
    .splice(0,k); 
```
화살표 함수는 중괄호가 없으면 해당 값을 리턴하니 이렇게 바꿔볼 수 있을 것 같습니다